### DISCUSSION TITLE
Automate obs

### DISCUSSION THREAD

**Discussion by shersoni610** [letmaik/pyvirtualcam#53] [Q&A]
Hello

As per the document one has to manually start virtual cam at least once and close obs. 

1.  Is there a way to automate the above step? I.e. get obs using curl command and start using it right way?

2. Do we need to have separate virtual camera for different resolutions? 1k, 2k, 4K.

3.  Which pixel format provides the best image quality? 

thanks for this very useful software.

**Comment by letmaik** [comment]
1. No.
2. You can only have a single virtual camera anyway, so no.
3. They are all fine, you will not visually see a difference.

**Reply by shersoni610** [reply]
Suppose I have to stream 4K images to a 1080p virtual camera then should I downsample before it is send or the virtual camera will take care of it.

thanks

**Comment by shersoni610** [comment]
If we are restricted to only one virtual camera then can we change the resolution and pixel format dynamically based on user specified applications?
